### PR TITLE
pw-top: add page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/pw-top.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`pw-top --batch-mode --iterations {{3}}`
+
+- Display help:
+
+`pw-top --help`
+
+- Display version:
+
+`pw-top --version`
```
Comment (cyqsimon) [inline pages/linux/pw-top.md:29]: Hmm. What's the policy on informational/conventional flags like these? I feel like they aren't particularly useful - if someone is using TLDR, I think it's fair to assume they likely know the existence of these conventional flags. But I do see them quite a bit in existing pages (`rg -- '--help'`).

**Review by cyqsimon — COMMENTED** [review]
Thanks for the contribution. I've got a minor suggestion.

File: pages/linux/pw-top.md (hunk 1 above)
Comment (kbdharun) [reply pages/linux/pw-top.md:29]: Agreed, we can remove them here. The help and version commands are particularly useful in Windows pages as there are a lot of differences/implementations for them depending upon the shell/utility.

File: pages/linux/pw-top.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# pw-top
+
+> View the pipewire nodes and devices statistics in real-time.
+> See also: `pipewire`, `pw-dump`, `pw-cli`, `pw-profiler`.
+> More information: <https://docs.pipewire.org/page_man_pw-top_1.html>.
+
+- Display an interactive view of pipewire nodes and devices:
```
Comment (kbdharun) [inline pages/linux/pw-top.md:7]:
```suggestion
> View the PipeWire nodes and devices statistics in real-time.
> See also: `pipewire`, `pw-dump`, `pw-cli`, `pw-profiler`.
> More information: <https://docs.pipewire.org/page_man_pw-top_1.html>.

- Display an interactive view of PipeWire nodes and devices:
```

File: pages.pt_BR/linux/pw-top.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# pw-top
+
+> Mostra os nós e estatísticas de dispositivos PipeWire em tempo real.
+> Veja também: `pipewire`, `pw-dump`, `pw-cli`, `pw-profiler`.
+> Mais informações: <https://docs.pipewire.org/page_man_pw-top_1.html>.
+
+- Mostra uma visualização interativa de nós e dispositivos pipewire:
```
Comment (kbdharun) [inline pages.pt_BR/linux/pw-top.md:7]:
```suggestion
- Mostra uma visualização interativa de nós e dispositivos PipeWire:
```

File: pages/linux/pw-top.md (hunk 1 above)
Comment (isaacvicente) [reply pages/linux/pw-top.md:29]: Yep, I think we should keep TLDR pages as minimal and useful as possible.
There's nothing wrong on removing these options (`--help` and `--version`) @vitorhcl.

**Review by isaacvicente — APPROVED** [review]

File: pages/linux/pw-top.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/linux/pw-top.md]: They are filler in some regard however there merit for newer users to Linux especially can be helpful. They are the first commands to be replace if better commands are needed, I think they should remain.

**Pull request merged**